File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
       int summa = 0;
       int n = nums.size();

       for(int i : nums)
           summa+= i;

       return (n * (n + 1))/2 - summa; // fórmula de sumatoria n * (n + 1) / 2

    }
};

File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0

        for i in nums:
            count += i

        rang = len(nums)

           
        return  ((rang * (rang + 1)) // 2) - count

        